object-store-client-play-26/src/test/scala/uk/gov/hmrc/objectstore/client/utils/DirectoryUtils.scala

```diff
@@ -16,14 +16,14 @@
 
 package uk.gov.hmrc.objectstore.client.utils
 
-import java.util.UUID
+import java.util.UUID.randomUUID
 
 import uk.gov.hmrc.objectstore.client.Path
 
 object DirectoryUtils {
-  def generateDirectoryPath(): Path.Directory =
-    Path.Directory(UUID.randomUUID().toString)
+  def directoryPath(path: String = randomUUID().toString): Path.Directory =
```

Comment:
I think we should have two functions here `generateDirectoryPath()` which delegates to `directoryPath(path: String)`
Reason is that when we don't pass a parameter, we get a random result - semantic indicated with the prefix `generate`.
Same for filePath.

Reply:
Do we actually need `directoryPath()` ? It's just an unnecessary level of indirection to using `Path.Directory()`

Reply:
Fixed